nodes/text/change_text_case.py

```diff
@@ -5,7 +5,8 @@
 
 caseTypeItems= [("UPPER", "To Upper Case", ""),
                 ("LOWER", "To Lower Case", ""),
-                ("CAPITALIZE", "Capitalize Phrase", "") ]
+                ("CAPITALIZE", "Capitalize Phrase", ""),
+                ("CAPITALIZE_WORDS", "Capitalize Words", "") ]
 
```

Comment:
oh and since you're working on it right now, can you also set the enum item values here? like "'NONE', 0)" ...